### ISSUE TITLE
VoiceOver does not indicate the difference between links and in-page links

### ISSUE THREAD

**Issue by NickColley** [alphagov/reported-bugs#42]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=198963
**Date:** 2019-07-18
**Reported by:** Nick Colley
**Related to:** https://github.com/alphagov/govuk-design-system/issues/902

---

### Overview

When an anchor link is read out to a user, it's not clear if this will take them to a new page or somewhere in the same page.

“ I found it highly challenging to understand which links were standard links and which would move my focus down the page as both were located within the header.”

### Expected result

Anchor links that jump to content within the page should have a suffix of 'same page link' as is consistent with JAWS

### Actual result

Anchor links that jump to content within the page do not indicate that they're in page links
